presto-main-base/src/main/java/com/facebook/presto/sessionpropertyproviders/NativeWorkerSessionPropertyProvider.java

```diff
@@ -357,6 +358,11 @@ public NativeWorkerSessionPropertyProvider(FeaturesConfig featuresConfig)
                         "Native Execution only. Priority of memory recliamer when deciding on memory pool to abort." +
                         "Lower value has higher priority and less likely to be choosen for memory pool abort",
                         2147483647,
+                        !nativeExecution),
+                integerProperty(
+                        NATIVE_MAX_NUM_SPLITS_LISTENED_TO,
+                        "Maximum number of splits to listen to per table scan node per worker.",
```

Comment:
Could you clarify what this listener does and I'm confused about `per table scan node per worker` statement